packages/types/src/order/mutations.ts

```diff
@@ -286,9 +288,9 @@ export interface ConfirmOrderChangeDTO {
 /** ORDER CHANGE ACTION START */
 
 export interface CreateOrderChangeActionDTO {
-  order_change_id: string
-  reference: string
-  reference_id: string
+  order_change_id?: string
+  reference?: string
+  reference_id?: string
```

Comment:
Q: we can potentially create change actions that are not attached?

Reply:
yes, they can be generic and open to any modification.